### PR TITLE
Revert Multi View Rendering

### PR DESCRIPTION
**Pull request by synoet**
It was decided, that for the time being the complexity introduced by multi view rendering is not worth the feature (Document Preview) for the time being.

### CODE REVIEW

**Review by whutchinson98 — APPROVED** [review]

File: qa-env/src/OfficeDocument/OfficeDocument.tsx [hunk 1]
```diff
@@ -273,7 +270,6 @@ export function OfficeDocument(props: Props) {
                   ? ZOOM_OUT_CANVAS_FIT
                   : ZOOM_IN_CANVAS_FIT,
                 'object-position': 'top center',
-                'image-rendering': 'crisp-edges',
```
Comment (chase) [inline qa-env/src/OfficeDocument/OfficeDocument.tsx:276]: Probably ideal to keep this

File: qa-env/src/OfficeDocument/OfficeDocument.tsx [hunk 2]
```diff
@@ -287,7 +283,6 @@ export function OfficeDocument(props: Props) {
...
```
Comment (chase) [inline qa-env/src/OfficeDocument/OfficeDocument.tsx:290]: And this

File: qa-env/src/OfficeDocument/OfficeDocument.tsx [hunk 3]
```diff
@@ -24,9 +24,8 @@ import { getOrCreateZoomSignal } from './zoom';
 import { getOrCreateDPISignal } from './twipConversion';
 
 // These give us good scaling behavior until the render actually finishes
-
-/** 'cover' is ideal for zoom_in, however it causes issues with pixelation. */
-const ZOOM_IN_CANVAS_FIT = 'contain';
```
Comment (chase) [inline qa-env/src/OfficeDocument/OfficeDocument.tsx:29]: Should we keep this?

**Review by chase — CHANGES_REQUESTED** [review]

**Comment by synoet** [comment]
@chase sorry github only showed me your comments after I squashed. I'll add this back in with the dpi fix.

**Pull request merged**